### DISCUSSION TITLE
Implement global Pointerevent prop for Stack.Screen

### DISCUSSION THREAD

**Discussion by EKT2000** [react-navigation/react-navigation#9953] [Ideas]
Hi, 

i wanted to ask if there is any possibility to implement a global pointerevent for Stack.Screen. I have a tab component which triggers underlying components from the screen. Therefore I am disabling the screen while the tab component is expanded. Right now I have to pass the pointer prop manually to all the screens, but it would be awesome to set this property directly on the navigation stack configuration.

**Comment by ViviZhang12** [comment]
I had a similar request, I use createNativeStackNavigator to implement a navigator,  I want the container(DebugContainer) doesn't handle touch events, but it's children handle the touch event, so if the touch event is not in the Stack.Screen's component scope but in the Container, it can be handled by the map view, which and the NavigationContainer are both in a view with layout type "absolute".
<img width="819" alt="Screen Shot 2022-04-15 at 2 29 26 PM" src="https://user-images.githubusercontent.com/103631846/163530917-c583630a-a5c4-4d5f-b618-c47f40691358.png">
I try to add pointerEvents to DebugContainer(in NativeStackView.native.tsx), and set it to 'box-none', it worked.
<img width="653" alt="Screen Shot 2022-04-15 at 2 33 57 PM" src="https://user-images.githubusercontent.com/103631846/163531243-afa5ef60-44ec-484a-88d1-c8d8330b0c13.png">

So is it possible to add a pointerEvents property to screenOptions?
